cves/CVE-2018-6621.yml

```diff
@@ -90,7 +94,7 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 118e1b0b3370dd1c0da442901b486689efd1654b
-  note: ''
+  note: The fix to this bug was very simple, just adding 1024LL to a check.
 - commit: ''
   note: ''
 vccs:
```

Comment:
Again mention the differences between the vccs. Upon a quick glance, it seems like this vulnerability was introduced when trying to fix a different issue.

Reply:
Do you mean in the section below fixes?

Reply:
I figured out what you meant.